Sources/SpelledPitch/PitchSpeller/Wetherfield/WeightedGraphSchemeProtocol.swift

```diff
@@ -38,51 +38,35 @@ extension WeightedGraphSchemeProtocol {
 extension WeightedGraphSchemeProtocol where Self: UndirectedGraphSchemeProtocol, Weight: Numeric {
     
     static func * (lhs: Self, rhs: Self) -> Self {
-        return Self { edge in
-            guard let lweight = lhs.weight(edge), let rweight = rhs.weight(edge) else { return nil }
-            return lweight * rweight
-        }
+        return Self { edge in lhs.weight(edge) ?? rhs.weight(edge) }
```

Comment:
Does this multiply if they are both non-nil ?

Reply:
Nope. Not one bit. V wrong.

Reply:
I actually ended up using this though for a (non-commutative) addition in the other branch...

Reply:
Very resourceful: "One person's trash is another person's treasure."